src/content/docs/extensions/algos/wcc.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Weakly Connected Components
+---
+
+A weakly connected component (WCC) in a directed graph is a maximal subgraph where any two vertices
+are connected to each other by some path, regardless of edge direction.
+
+Kuzu implements a parallelized version of the WCC algorithm based on [Ligra](https://jshun.csail.mit.edu/ligra.shtml).
+
+## Syntax
+
+```cypher
+CALL weakly_connected_components(
+    graph_name,
+    maxIterations := 100
+)
+RETURN node, group_id
+```
+
+#### Parameters
+
+| Parameter | Type | Optional | Default | Description |
+|-----------|------|----------|---------|-------------|
+| graph_name | STRING | No | - | Name of the projected graph to run the algorithm on |
+| maxIterations | INT64 | Yes | 100 | Maximum number of iterations to run |
+
+
+#### Return
+
+| Column | Type | Description |
+|--------|------|-------------|
+| node | NODE | Node object |
+| group_id | INT64 | The group ID that the node belongs to |
+
+
+## Example
```

Comment:
It would be better if we can provide a visualization of the graph for illustration here.

Reply:
Let's discuss what kind of visualization we want. Explorer or other tools.

Reply:
Explorer is good enough, we can just use that.